### PR TITLE
fix: include box numbers in postal code changed and corrected events

### PR DESCRIPTION
**Pull request by jvandaal**

### CODE REVIEW

File: src/AddressRegistry/StreetName/Events/AddressPostalCodeWasChangedV2.cs [hunk 1]
```diff
@@ -21,6 +21,9 @@ public class AddressPostalCodeWasChangedV2 : IStreetNameEvent, IHasAddressPersis
         [EventPropertyDescription("Objectidentificator van het adres.")]
         public int AddressPersistentLocalId { get; }
 
+        [EventPropertyDescription("Objectidentificatoren van de gekoppelde busnummers.")]
+        public List<int> BoxNumberPersistentLocalIds { get; }
```
Comment (lvermeulen) [inline src/AddressRegistry/StreetName/Events/AddressPostalCodeWasChangedV2.cs:25]: we should prefer IEnumerable<T>, ICollection<T> or IList<T> -- in this case the consumer just needs an IEnumerable<T>, no?

File: src/AddressRegistry/StreetName/Events/AddressPostalCodeWasCorrectedV2.cs [hunk 2]
```diff
@@ -21,6 +21,9 @@ public class AddressPostalCodeWasCorrectedV2 : IStreetNameEvent, IHasAddressPers
         [EventPropertyDescription("Objectidentificator van het adres.")]
         public int AddressPersistentLocalId { get; }
 
+        [EventPropertyDescription("Objectidentificatoren van de gekoppelde busnummers.")]
+        public List<int> BoxNumberPersistentLocalIds { get; }
```
Comment (lvermeulen) [inline src/AddressRegistry/StreetName/Events/AddressPostalCodeWasCorrectedV2.cs:25]: same remark

File: src/AddressRegistry/StreetName/Events/AddressPostalCodeWasChangedV2.cs (hunk 1 above)
Comment (jvandaal) [reply src/AddressRegistry/StreetName/Events/AddressPostalCodeWasChangedV2.cs]: Changed.

File: src/AddressRegistry/StreetName/Events/AddressPostalCodeWasCorrectedV2.cs (hunk 2 above)
Comment (jvandaal) [reply src/AddressRegistry/StreetName/Events/AddressPostalCodeWasCorrectedV2.cs]: Changed.

**Pull request merged**